HelpSource/Classes/ArrayedCollection.schelp

```diff
@@ -68,37 +68,119 @@ code::
 
 method::at
 Return the item at strong::index::.
-
-The index can also be an Array of indices to extract specified elements. Example:
+The index can also be an array of indices to extract the specified elements.
+link::Classes/Array:: is its syntactic shortcut.
 code::
-x = [10, 20, 30];
-y = [0, 0, 2, 2, 1];
-x[y]; // returns [10, 10, 30, 30, 20]
+x = [1, 2, 3] //-> [1, 2, 3]
+
+// Retrieve the value at index 1:
+x.at(1) //-> 2
+// Equivalent shorthand syntax:
+x[1] //-> 2
+
+// An index can also be retrieved from an array:
+y = [0, 0, 2, 2, 1] //-> [0, 0, 2, 2, 1]
+x[y]; //-> [1, 1, 3, 3, 2]
+
+(
+var pitches =  [43.0, 55.0, 58.86, 57.42, 67.0];
+s.waitForBoot {
```

Comment:
there is a general assumption in all the helfiles that the server is booted.

Reply:
I believe that `s.waitForBoot` is more user-friendly; however, I changed it according to your judgment.